packages/wrangler/src/versions/upload.ts

```diff
@@ -422,15 +427,6 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 			try {
 				const body = createWorkerUploadForm(worker);
 
-				body.set(
```

Comment:
Is this a breaking change for current users?

Reply:
It's a bug fix, so it changes behavior that message was ignored and now it's actually set. But version-upload command is gated both by the `--experimental-gradual-rollouts` wrangler flag as well at gated at the server level, so we're ok to make breaking changes here if needed currently.

Reply:
It's not a breaking change. The API wasn't expecting annotations there but under metadata instead – so it's a fix.

Reply:
Ah, I didn't realise this wasn't in the wrangler-wide form data creation